week5/test react app/sha-test-app/src/components/selectedFriend/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import "./styles.css";
+import FriendDetails from "./../FriendDetails";
+
+function SelectedFriend(props) {
+  let [hidden, setHidden] = useState(true);
+
+  function toggleHidden() {
+    setHidden((hidden = !hidden));
```

Comment:
This is an assignment inside the setHidden function. The correct should be
```suggestion
    setHidden(!hidden);
```